src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Utf8Parser/Utf8Parser.Boolean.cs

```diff
@@ -12,16 +12,14 @@ public static partial class Utf8Parser
         /// </summary>
         /// <param name="source">The Utf8 string to parse</param>
         /// <param name="value">Receives the parsed value</param>
-        /// <param name="bytesConsumed">On a successful parse, receives the length in bytes of the substring that was parsed </param>
-        /// <param name="standardFormat">Expected format of the Utf8 string</param>
+        /// <param name="bytesConsumed">On a successful parse, receives the length in bytes of the substring that was parsed</param>
+        /// <param name="standardFormat">Expected format of the Utf8 string. Supported formats are G and l.</param>
```

Comment:
A clarification on formats has been added because the current implementation checks the format without using it afterwards.

https://github.com/dotnet/runtime/blob/8e92aef5387fe1d4b9159b4a3657416ac7d0a05a/src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Utf8Parser/Utf8Parser.Boolean.cs#L31-L32

Should this check be removed?

Reply:
Ok. I won't change this validation.